### PR TITLE
Add release notes for Katello 4.11

### PR DESCRIPTION
**Pull request by ianballou**
Release notes for Katello 4.11.0.

### CODE REVIEW

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+=== Ansible Collections
+
+* pass:[Cannot sync ansible collections repositories to smart proxy] - https://projects.theforeman.org/issues/35746[#35746]
+
+=== Container
+
+* pass:[Taxonomy filtration on Container Image Tags page does not work as expected] - https://projects.theforeman.org/issues/36911[#36911]
+* pass:[ Cannot discovery container repositories on private registries or on registries that only support api v2] - https://projects.theforeman.org/issues/36861[#36861]
+* pass:[Container registries for Sat and Capsule set wrong token expiration field] - https://projects.theforeman.org/issues/36827[#36827]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello-4.11.0.adoc:26]:
```suggestion
* pass:[Container registries for Sat and Capsule set wrong token expiration field] - https://projects.theforeman.org/issues/36827[#36827]
```

Please use either "Foreman Server" and "Smart Proxy Server", or "{ProjectServer}" and "{SmartProxyServer}" if possible.

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+=== Content Views
+
+* pass:[Satellite showing the wrong date when using a filter when the 'end date'] - https://projects.theforeman.org/issues/36883[#36883]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello-4.11.0.adoc:30]: Foreman Server/"{ProjectServer}"

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+* pass:[Promoting a composite content view to environment with registry name as "<%= lifecycle_environment.label %>/<%= repository.name %>" on Red Hat Satellite 6 fails with "'undefined method '#label' for NilClass::Jail (NilClass)'"] - https://projects.theforeman.org/issues/36776[#36776]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello-4.11.0.adoc:31]: "{ProjectServer}"?

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+* pass:[Bring back duplicate content warning for composite CVs for non-docker repos] - https://projects.theforeman.org/issues/36492[#36492]
+* pass:[Allow searching for composite content views that contain a content view] - https://projects.theforeman.org/issues/35901[#35901]
+
+=== Errata Management
+
+* pass:[Generate applicability tasks fails with error "ERROR:  insert or update on table "katello_content_facet_errata" violates foreign key constraint "katello_content_facet_errata_ca_id"] - https://projects.theforeman.org/issues/36914[#36914]
+* pass:[errata's issued and updated times shouldn't be changed to local timezone on WebUI] - https://projects.theforeman.org/issues/36882[#36882]
+* pass:[Timeout for "hammer  --no-headers erratum list --errata-restrict-applicable 1 --organization-id 1"] - https://projects.theforeman.org/issues/36835[#36835]
+* pass:[Applied Errata report download fails with undefined method `value' for nil:NilClass error] - https://projects.theforeman.org/issues/36811[#36811]
+* pass:[Recalculate button for Errata is not available on Satellite 6.13/ Satellite 6.14 if no errata is present] - https://projects.theforeman.org/issues/36790[#36790]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello-4.11.0.adoc:42]: Please use upstream terminology.

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+=== Foreman Proxy Content
+
+* pass:[Store all env_ids for smart_proxy complete sync in task input or output] - https://projects.theforeman.org/issues/36873[#36873]
+* pass:[Trigger capsule content count update action after Orphan cleanup] - https://projects.theforeman.org/issues/36857[#36857]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello-4.11.0.adoc:47]:
```suggestion
* pass:[Trigger {SmartProxyServer} content count update action after Orphan cleanup] - https://projects.theforeman.org/issues/36857[#36857]
```

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+* pass:[Last capsule sync date should have a way to query per environment] - https://projects.theforeman.org/issues/36852[#36852]
+* pass:[Properly translate rpm.modulemd to "module_stream"] - https://projects.theforeman.org/issues/36820[#36820]
+* pass:[As a user, I want to have a way to refresh the calculated counts on-demand on the UI] - https://projects.theforeman.org/issues/36807[#36807]
+* pass:[Delete oprhan content task doesn't remove orphaned remotes in the Capsule] - https://projects.theforeman.org/issues/36787[#36787]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello-4.11.0.adoc:51]:
```suggestion
* pass:[Delete oprhan content task doesn't remove orphaned remotes in {SmartProxyServer}] - https://projects.theforeman.org/issues/36787[#36787]
```

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 7]
```diff
@@ -4,14 +4,48 @@
...
+
+Features performance improvements most notably around import/export for disconnected environments.
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+=== Pulpcore
+
+The pulpcore module stream must be disabled before updating packages.  Pulpcore RPMs are no longer modular.
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello.adoc:44]:
```suggestion
The pulpcore module stream must be disabled before updating packages.
Pulpcore RPMs are no longer modular.
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
Please have a look at the branding and use attributes _if possible_*:

* {ProjectServer}
* {SmartProxyServer}
* {ProjectWebUI}

this should cover most of the release notes.

\* Disclaimer: I don't know who "pass:[]" and attributes work together.

**Comment by ianballou** [comment]
@maximiliankolb the changelog entries in ([guides/doc-Release_Notes/topics/katello-4.11.0.adoc](https://github.com/theforeman/foreman-documentation/pull/2632/files/046b96d6e0c4585e7c5b021248999263de88373c#diff-92085ffa94f6df37622dc0164df481ec42467339f5e59ecddd1956d57d3f3ac8)) are made to mirror what we have here: https://github.com/Katello/katello/blob/KATELLO-4.11/CHANGELOG.md


I see there are some specific Capsule & Satellite references that people failed to sanitize out of BZ->redmine syncs, so I'll fix those in redmine.

**Comment by ianballou** [comment]
@maximiliankolb everything here should be in upstream terminology now. Had to update a bunch of Redmines. For GA, the Katello changelog will be update to match.

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 8]
```diff
@@ -4,14 +4,48 @@
...
 
 [id="katello-deprecations"]
 == Deprecations
 
-There are no deprecations with Katello {KatelloVersion}.
+=== Simple Content Access (SCA)
```
Comment (jeremylenz) [inline guides/doc-Release_Notes/topics/katello.adoc:49]: It's this that's deprecated, not SCA :)

```suggestion
=== Entitlement-based subscription management
```

**Review by jeremylenz — COMMENTED** [review]
Headline features LGTM, just one comment!

**Review by jeremylenz — APPROVED** [review]

**Comment by ianballou** [comment]
I don't think the lint errors are related.

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+* pass:[Remove deprecated docker_tags_whitelist] - https://projects.theforeman.org/issues/36695[#36695]
+* pass:[Remove katello-agent API endpoints] - https://projects.theforeman.org/issues/36665[#36665]
+
+=== Activation Key
+
+* pass:[Add system purpose card to new AK details page] - https://projects.theforeman.org/issues/36610[#36610]
+
+=== Ansible Collections
+
+* pass:[Cannot sync ansible collections repositories to smart proxy] - https://projects.theforeman.org/issues/35746[#35746]
```
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/doc-Release_Notes/topics/katello-4.11.0.adoc:20]: I'd avoid that. They are the titles of Redmine issues and they should match. Plus, we don't do any branding on these (upstream) release notes anyway.

File: guides/doc-Release_Notes/topics/katello-4.11.0.adoc [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+* pass:[Notification via email when repo sync fail] - https://projects.theforeman.org/issues/17748[#17748]
+
+=== API
+
+* pass:[Remove deprecated docker_tags_whitelist] - https://projects.theforeman.org/issues/36695[#36695]
+* pass:[Remove katello-agent API endpoints] - https://projects.theforeman.org/issues/36665[#36665]
+
+=== Activation Key
+
+* pass:[Add system purpose card to new AK details page] - https://projects.theforeman.org/issues/36610[#36610]
```
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/doc-Release_Notes/topics/katello-4.11.0.adoc:16]: I updated the Redmine issue so the next time this file is regenerated, it should be fixed.

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 11]
```diff
@@ -4,14 +4,49 @@
...
+Features performance improvements most notably around import/export for disconnected environments.
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+=== Pulpcore
+
+The pulpcore module stream must be disabled before updating packages. 
+Pulpcore RPMs are no longer modular.
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:45]: While upgrading, users must disable the module

**Review by ekohl — APPROVED** [review]
I'm merging this now, but we should iterate on it for GA. Next time, this should be merged even before the RC is announced.

**Pull request merged**